taichi/transforms/type_check.cpp

```diff
@@ -412,7 +412,11 @@ namespace irpass {
 
 void typecheck(IRNode *root) {
   check_fields_registered(root);
-  return TypeCheck::run(root);
+  TypeCheck::run(root);
+  //  if (root->is<Block>() && root->as<Block>()->parent == nullptr) {
```

Comment:
Are these comments necessary here?

Reply:
I think we can `verify` after each `typecheck`. However, 23 tests are failing if I uncomment this now...

Reply:
Sounds good - I'll merge this in once CI passes. Let's fix other issues later in separate PRs. Thanks so much!!